src/main/java/seedu/address/model/quiz/QuizResult.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.quiz;
+
+import java.util.Objects;
+
+import seedu.address.model.question.Answer;
+import seedu.address.model.question.QuestionBody;
+
+/**
+ * Represents a question. Its answer, questionBody, quizTime and result are guaranteed non-null.
+ */
+public class QuizResult {
+    private final Answer answer;
+    private final QuestionBody questionBody;
+    private final String quizTime;
+    private final boolean result;
+
+    public QuizResult(Answer answer, QuestionBody questionBody, String quizTime, boolean result) {
+        this.answer = answer;
+        this.questionBody = questionBody;
+        this.quizTime = quizTime;
+        this.result = result;
+    }
+
+    public Answer getAnswer() {
+        return answer;
+    }
+
+    public QuestionBody getQuestionBody() {
+        return questionBody;
+    }
+
+    public String getQuizTime() {
+        return quizTime;
+    }
+
+    public boolean getResult() {
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        if (!(other instanceof QuizResult)) {
+            return false;
+        }
+
+        QuizResult otherQuizResult = (QuizResult) other;
+        return otherQuizResult.getQuestionBody().equals(getQuestionBody());
```

Comment:
You _can_ have duplicate question bodies in the quiz results – I was getting the unnecessary error `Quiz result contains duplicate results.`

This line should be `return otherQuizResult.getQuizTime().equals(getQuizTime());`.

Reply:
Sure. I will modify that. Thank you very much for the help. For the display question I will create a new box for it, so I will just temporarily leave it at fourth column for v1.2 and I will solve it in v1.3.

Reply:
Sure, I will modify that. Thank you very much for the help. For the display of quiz questions, I will create a new Ui window to display all quiz questions. So I will leave it at fourth column temporarily for v1.2 and implement further on v1.3.